Mikado/Mikado/Controller/TripTableViewController.swift

```diff
@@ -11,36 +11,47 @@ import UIKit
 class TripTableViewController: UITableViewController {
     
     var trips : [Trip]?
```

Comment:
You can make this non-optional, not sure why it's optional in the first place but it should make for less unwrapping :)